services/dask-sidecar/src/simcore_service_dask_sidecar/_meta.py

```diff
@@ -0,0 +1,37 @@
+""" Application's metadata
+
+"""
+
+
+from typing import Final
+
+import dask
+from packaging.version import Version
+from servicelib.utils_meta import PackageInfo
+
+info: Final = PackageInfo(package_name="simcore_service_dask_sidecar")
+__version__: Final[str] = info.__version__
+
+
+PROJECT_NAME: Final[str] = info.project_name
+VERSION: Final[Version] = info.version
+API_VERSION: Final[str] = info.__version__
+API_VTAG: Final[str] = info.api_prefix_path_tag
+SUMMARY: Final[str] = info.get_summary()
+
+# https://patorjk.com/software/taag/#p=display&f=Standard&t=dask%20sidecar
+APP_STARTED_BANNER_MSG = rf"""
+      _           _          _     _
```

Comment:
How come you rename this? how does that now works in tasks.py??? I do not see any modification there?
Also I am not sure I see the point of adding the ```_``` in front since this is a very simple process and the print_banner is imported in tasks.py.